cmd/uplink/cmd/cp.go

```diff
@@ -88,6 +90,17 @@ func upload(ctx context.Context, bs buckets.Store, srcFile string, destObj *url.
 		return err
 	}
 
+	/* create a signal of type os.Signal */
+	c := make(chan os.Signal, 0x01)
+
+	/* register for the os signals */
+	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I can see that there is another registration for these signals in `process/exec_conf.go`, so this one here is redundant. Plus no one is reading from the `c` channel. Perhaps, we can just remove this.

Reply:
oops... missed out on clean up...  ..will remove it.. it shouldnt be here. Will fix this... but  is my understanding right? that c chnl is never created as there is no a reader of that chnl? I started to like GO...

Reply:
The channel is created on line 94. The fact that there is no one to read from it does not make it non existing.